test/rpc/handlers/get-value.spec.js

```diff
@@ -87,6 +87,7 @@ describe('rpc - handlers - GetValue', () => {
 
       const msg = new Message(T, key, 0)
 
+      dht.peerStore.put(other)
```

Comment:
Did this need to get added?

Reply:
Yes, in the code we try to find the peer in the `peer-store`. After this test we have another one for unkwown peers